offlineimap.conf

```diff
@@ -432,6 +432,62 @@ remoterepository = RemoteExample
 #proxy = SOCKS5:IP:9999
 
 
+# This option stands in the [Account Test] section.
+#
+# IMAP defines an encoding for non-ASCII ("international") characters, and most
+# IMAP servers store folder names in this encoding. Note that the IMAP 4rev1
+# specification (RFC 3501) allows both UTF-8 and modified UTF-7 folder names
+# so it is *possible* that an IMAP server already uses UTF-8 encoded folder
+# names. But usually Folders that are shown as, say, "Gäste" will be represented
+# as "G&AOQ-ste", and by default will be synchronized like this by offlineIMAP.
+#
+# This option converts IMAP folder names from IMAP4-UTF-7 to UTF-8 and back
+# in order to have nicely readable UTF-8 folder names in the local copy.
+#
+# WARNING: with this option enabled:
+# - compatibility with any other version is NOT GUARANTED (including newer);
+# - no support is provided.
+#
+# This feature is EXPERIMENTAL and was merged because it's small changes in the
+# code.  However, this might seriously decrease the stability of the program.
```

Comment:
Please, remove "and was merged because...". This was true for decodefoldernames that I never intented to mark stable. 

However, this news feature is the correct way to implement encodings and if this proves to be stable I do intend to mark it as TESTING and finally STABLE.

Reply:
Same goes for the next sentence.

Reply:
(below)